File: vue-vben-admin/apps/web-naive/src/api/entity/SysLoginLog.ts
export interface SysLoginLog {
  userId: string  | null | undefined ;    //  用户ID
  userName: string  | null | undefined ;    //  用户名
  userCode: string  | null | undefined ;    //  用户编码
  loginTime: string  | null | undefined ;    //  登录时间
  loginIp: string  | null | undefined ;    //  登录IP地址
  device: string  | null | undefined ;    //  登录设备
  browser: string  | null | undefined ;    //  浏览器信息
  errorMsg: string  | null | undefined ;    //  错误信息
  screenResolution: string  | null | undefined ;    //  登录设备分辨率
  sessionId: string  | null | undefined ;    //  会话ID
  tenantId: string  | null | undefined ;    //  租户GUID
  id: string  | null | undefined ;    //  主键
  createdId: string  | null | undefined ;    //  创建人GUID
  createdTime: string  | null | undefined ;    //  创建时间
  creator: string  | null | undefined ;    //  创建人
  updatedTime: string  | null | undefined ;    //  更新时间
  updatedId: string  | null | undefined ;    //  更新人GUID
  updator: string  | null | undefined ;    //  更新人
}

File: src/SimpleApp/Simple.AdminApplication/SysMng/Entities/SysLoginLog.cs
namespace Simple.AdminApplication.SysMng.Entities;

/// <summary>
/// 登陆日志
/// </summary>
public class SysLoginLog : DefaultTenantEntity
{
    /// <summary>
    /// 用户ID
    /// </summary>
    public Guid? UserId { get; set; }

    /// <summary>
    /// 用户名
    /// </summary>
    public string UserName { get; set; }
    
    /// <summary>
    /// 用户编码
    /// </summary>
    public string UserCode { get; set; }

    /// <summary>
    /// 登录时间
    /// </summary>
    public DateTime LoginTime { get; set; } 

    /// <summary>
    /// 登录IP地址
    /// </summary>
    public string LoginIp { get; set; }

    /// <summary>
    /// 登录设备
    /// </summary>
    public string Device { get; set; }

    /// <summary>
    /// 浏览器信息
    /// </summary>
    public string Browser { get; set; }

    /// <summary>
    /// 错误信息
    /// </summary>
    public string ErrorMsg { get; set; }

    /// <summary>
    /// 登录设备分辨率
    /// </summary>
    public string ScreenResolution { get; set; }

    /// <summary>
    /// 会话ID
    /// </summary>
    public Guid? SessionId { get; set; }
}